### PR TITLE
webutil.useradd throws exception KeyError: 'out'

### PR DESCRIPTION
**Pull request by gweis**
return dictionary from cmd.run_all directly.

### CODE REVIEW

File: salt/modules/htpasswd.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def useradd(pwfile, user, password, opts=''):
         salt '*' webutil.useradd /etc/httpd/htpasswd larry badpassword
         salt '*' webutil.useradd /etc/httpd/htpasswd larry badpass opts=ns
     '''
-    return useradd_all(pwfile, user, password, opts=opts)['out'].splitlines()
+    return useradd_all(pwfile, user, password, opts=opts)
```
Comment (thatch45) [inline salt/modules/htpasswd.py]: The line should be:

```
return useradd_all(pwfile, user, password, opts=opts)['stdout'].splitlines()
```

Can you please check that this line works and fix accordingly?

**Comment by gweis** [comment]
Yes, that was what I have thought at first as well, but in case the htpasswd command fails it won't return any error status if you do it that way.

**Comment by thatch45** [comment]
now that I have merged it I just realized that we should give a more custom return based on the data from the command since this command obviously never worked

**Comment by gweis** [comment]
Well, it worked until this commit 2d868bb29ddf3ab5de79348e34e0176e8ad9c1fe made it into the release.

Yes it would be good to know what a module function is supposed to return. Took me a while to figure out that returning the dict directly does what I wanted.

**Pull request merged**